docs/contributor/draft/rate-limit-design.md

```diff
@@ -0,0 +1,80 @@
+# Rate Limit Proposal
+
+## Spec 
```

Comment:
Where is the `context` field in the spec?

Reply:
just added, thanks!